Framework/DataHandling/test/LoadILLReflectometryTest.h

```diff
@@ -86,7 +87,7 @@ class LoadILLReflectometryTest : public CxxTest::TestSuite {
     for (const auto &p : properties) {
       loader.setPropertyValue(p.first, p.second);
     }
-    TS_ASSERT_THROWS_NOTHING(loader.execute(););
+    TS_ASSERT_THROWS_NOTHING(loader.execute());
```

Comment:
Some semicolons `;` have appeared in after the asserts. These should not be required as `TS_ASSERT_THROWS_NOTHING` are macros.